libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/audioguidance/AppAudioGuidanceUi.kt

```diff
@@ -45,15 +45,15 @@ fun Lifecycle.muteAudioGuidance() {
     addObserver(object : DefaultLifecycleObserver {
         lateinit var initialState: MapboxAudioGuidanceState
         override fun onResume(owner: LifecycleOwner) {
-            with(MapboxAudioGuidance.getInstance()) {
+            with(MapboxAudioGuidance.getRegisteredInstance()) {
                 initialState = stateFlow().value
                 mute()
             }
         }
 
         override fun onPause(owner: LifecycleOwner) {
             if (!initialState.isMuted) {
-                MapboxAudioGuidance.getInstance().unMute()
+                MapboxAudioGuidance.getRegisteredInstance().unMute()
```

Comment:
Can we also rename `unMute` to `unmute`? It's one word, not two. Also the function in `MapboxSoundButton` is named `unmute`.

Reply:
maybe this was the reason it was called `unMute`

![Screen Shot 2022-10-06 at 7 01 29 AM](https://user-images.githubusercontent.com/3021882/194333315-35082e20-71a2-40e9-bc3d-eec0cf638d87.png)

Reply:
I think we can ignore this warning, because unmute is an actual word: https://dictionary.cambridge.org/dictionary/english/unmute.

Reply:
fine by me